src/SprykerSdk/Zed/AopSdk/Business/OpenApi/Builder/OpenApiCodeBuilder.php

```diff
@@ -0,0 +1,732 @@
+        }
+
+        if ($path === '') { //Set error message
+            $messageTransfer = new MessageTransfer();
+            $messageTransfer->setMessage(sprintf('Module name not found for path %s', $path));
+            $this->openApiResponseTransfer->addError($messageTransfer);
+
+            return '';
+        }
+
+        $pathFragments = explode('/', trim($path, '/'));
+
+        return sprintf(ucwords(current($pathFragments)) . '%s', 'Api');
+    }
+
+    /**
+     * @param string $path
+     * @param \cebe\openapi\spec\Operation $operation
+     *
+     * @return string
+     */
+    protected function getControllerName(string $path, Operation $operation): string
+    {
+        if (isset($operation->operationId)) {
+            $operationId = explode('.', $operation->operationId);
+
+            return $this->inflector->classify(sprintf(end($operationId) . '%s', 'Controller'));
+        }
+
+        $pathFragments = explode('/', trim($path, '/'));
+        foreach (array_reverse($pathFragments) as $key => $resource) {
+            if ($key === (count($pathFragments) - 1)) {
+                $resource = sprintf($resource . '%s', 'Resource');
+            }
+
+            if (strpos($resource, '{') === false) {
+                return $this->inflector->classify(sprintf("{$resource}%s", 'Controller'));
+            }
+        }
+
+        $messageTransfer = new MessageTransfer();
+        $messageTransfer->setMessage(sprintf('Controller name not found for path %s', $path));
+        $this->openApiResponseTransfer->addError($messageTransfer);
+
+        return '';
+    }
+
+    /**
+     * @param string $organization
+     * @param \cebe\openapi\spec\OpenApi $openApi
+     *
+     * @return void
+     */
+    protected function generateTransfers(
+        string $organization,
+        OpenApi $openApi
+    ): void {
+        $transferDefinitions = [];
+        /** @var \cebe\openapi\spec\PathItem $pathItem */
+        foreach ($openApi->paths->getPaths() as $path => $pathItem) {
+            $transferDefinitions[$path] = $this->getTransferDefinitionFromPathItem($path, $pathItem);
+        }
+
+        if ($this->openApiResponseTransfer->getErrors()->count() === 0) {
+            $transferBuildSprykCommands = $this->getTransferDefinitionSprykCommands($organization, $transferDefinitions);
+            $this->runCommands($transferBuildSprykCommands);
+        }
+    }
+
+    /**
+     * @param string $path
+     * @param \cebe\openapi\spec\PathItem $pathItem
+     *
+     * @return array
+     */
+    protected function getTransferDefinitionFromPathItem(string $path, PathItem $pathItem): array
+    {
+        $transferDefinitions = [];
+
+        /** @var \cebe\openapi\spec\Operation $operation */
+        foreach ($pathItem->getOperations() as $method => $operation) {
+            $transferDefinitions[$method]['controllerName'] = $this->getControllerName($path, $operation);
```

Comment:
This is not as I explained it. `$method` and `controllerName` are not relevant for transfer generation. Those are only used with the upcoming steps.